CruzHacks/cruzhacks-2021-website
# Mission section

Problem
=======
As a prospective hacker,
I would like to read CruzHacks' mission statement
To understand more about the organization and event.

[Link to Asana Ticket](https://app.asana.com/1234)

Solution
========
To implement the `Mission` section of the homepage, flexbox-based positioning, (negative) margins, and other CSS styling techniques were used to replicate the provided Figma design. Text color variables and other box variations will be replaced after pre-existing pull requests are merged in. In the `Home` component, `MissionProps` corresponding to dictionary objects that represent each title and body text are then passed into the corresponding component as input parameters.

Change summary:
---------------
* finished declaring + styling mission statements f85dff0

Steps to Verify:
----------------
1. Checkout branch using `git checkout feature/mission`
2. Start local development server using `yarn start`
3. Verify that mission section is designed and responds correctly.
